### PR TITLE
[ICDS] month filter in SSD report start from Feb 2018

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @calellowitz @Rohit25negi 

I changed the month filter for SDD report and now we can choose only dates after Feb 2018. Also, I added info message when the user loading the report and have chosen date before Feb 2018.

Need QA: no
Environment: ICDS
Locally tested: yes

Regards,
Łukasz

### CODE REVIEW

**Review by calellowitz — COMMENTED** [review]
Is there a ticket this is from? The code looks fine but I am not super happy with redirecting the user to the current month. That seems quite surprising to the user. I am hoping to figure out the rationale for that and if there might be a better option. Can you also include a screenshot of the warning message?

**Comment by lwyszomi** [comment]
Hi @calellowitz Sorry, I forgot to add ticket: https://dimagi-dev.atlassian.net/browse/ICDS-644

Please don't merge this I found one case where this not working. I will update the PR as soon as possible.

**Comment by lwyszomi** [comment]
Hi @calellowitz I updated the PR, now all work correctly. Also, I'm adding the screenshot:
info_message_sdd

File: custom/icds_reports/static/js/directives/month-filter/month-filter.directive.js
Comment (Rohit25negi) [inline custom/icds_reports/static/js/directives/month-filter/month-filter.directive.js]: This should be starting from feb 2019 not feb -2018 as ticket says.

File: custom/icds_reports/templates/icds_reports/icds_app/service-delivery-dashboard.html
Comment (Rohit25negi) [inline custom/icds_reports/templates/icds_reports/icds_app/service-delivery-dashboard.html]: Feb 2019

File: custom/icds_reports/static/js/directives/navigation/navigation.directive.js
Comment (Rohit25negi) [inline custom/icds_reports/static/js/directives/navigation/navigation.directive.js]: Will it change the query param in the url also? or its just a ajax request?

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Left Some comments to correct the filter. and one general question.

File: custom/icds_reports/static/js/directives/navigation/navigation.directive.js
Comment (lwyszomi) [reply custom/icds_reports/static/js/directives/navigation/navigation.directive.js]: The params will be changed in both locations.

**Review by Rohit25negi — APPROVED** [review]
Looks good to me. Good to merge once the test cases passes

**Comment by calellowitz** [comment]
Made the same note in teh revert PR, but just to make sure it is seen, I reverted this because the ticket says to prompt the user to select a month, not redirect to the current month. I agree with the ticket that prompting the user to select is a better UI so am reverting this change until that is implemented

**Pull request merged**
